Packs/SpyCloudPack/README.md

```diff
@@ -0,0 +1,9 @@
+# SpyCloud Integration
+Integration for the SpyCloud ATO service. Retrieve breach data for
+- Domain searches
+- Email searches
+- Breaches
+
+Filter arguments are added as well. 
+
+**Make sure to whitelist your outbound IP address XSOAR uses to connect**
```

Comment:
```suggestion
```

When the pack is only of an integration (as in this case), we can use only the integration readme (no need for duplicate).